### ISSUE TITLE
DecksetParser breaks Presentation default shortcodes

### ISSUE THREAD

**Issue by ulfben** [OleVik/grav-plugin-presentation-deckset#2]
This might be by design, in which case I apologize for the noise. However; I was surprised to learn than setting Custom Parser to DecksetParser broke default functionality in the Presentation-plugin. For example:

```
[data-background-image=image.jpg]
[data-background-size=contain]
[style-background-color=transparent]
```
These will render as plain-text if DecksetParser is used. Is that intentional?

**Comment by OleVik** [comment]
It is not, entirely, as this addon should not have to override that parent method. I'll check and see why.

**Comment by OleVik** [comment]
This will be fixed in the upcoming v1.2.0.
